### ISSUE TITLE
Invert the current operation of Holes in Regions to affect layers below the Hole layer rather than those above

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#10721]
It may be more intuitive to have the hole apply to any region that is below it rather than the layers above it. (This would be similar to the way ‘mask’ and effect layers apply in photoshop or other art software)

**Comment by anathemamask** [comment]
Secondarily consider indenting the layers to indicate affected layers, mockup from photoshop courtesy of @unsoluble

![image](https://github.com/foundryvtt/foundryvtt/assets/26660743/0b1e8a81-52d9-4510-8874-3c25d46456f9)

**Comment by aaclayton** [comment]
Needs discussion. Also for the sake of precision I believe what @anathemamask means when using the term "layer" is **shape**.

**Comment by anathemamask** [comment]
Corrected terminology, sorry!

**Comment by Norc** [comment]
Currently, when you create some Shapes and then add a Shape that is a hole, the hole is placed at the bottom of the Shapes dialog and therefore "punches through" everything above it, which is good and results in a behavior that is expected when creating a new Shape.

If this was implemented, the new Hole Shape would have to be auto-inserted at the top of the dialog instead.

Playing around with this, the current behavior feels fine to me, but I don't have deep experience in Photoshop and similar software either.

**Comment by unsoluble** [comment]
Yeah while it makes _internally consistent_ sense in the current design, it's the opposite of how (I think literally) every design app works. :)

**Comment by aaclayton** [comment]
I think there are reasonable people who would reasonably have different opinions on this topic.
